src/dex/aave-v3-stata-v2/config.ts

```diff
@@ -0,0 +1,28 @@
+import { DexParams } from './types';
+import { DexConfigMap, AdapterMappings } from '../../types';
+import { Network, SwapSide } from '../../constants';
+import {
+  AaveV3Ethereum,
+  AaveV3EthereumLido,
+  AaveV3Gnosis,
+} from '@bgd-labs/aave-address-book';
+
+export const AaveV3StataConfig: DexConfigMap<DexParams> = {
+  AaveV3Stata: {
+    [Network.MAINNET]: {
+      factoryAddresses: [
+        AaveV3Ethereum.STATA_FACTORY,
```

Comment:
there is one factory per pool, so can be multiple per network (prev implementation did not support that)